src/auth/ui/vue/show.ts

```diff
@@ -236,7 +244,11 @@ export class AuthWebview extends VueWebview {
                 return { id: 'badStartUrl', text: `Connection failed. Please verify your start URL.` }
             }
 
-            getLogger().error('AuthWebview: Failed to setup: %s', (e as Error).message)
+            // If SSO setup fails we want to be able to show the user an error in the UI, due to this we cannot
+            // throw an error here. So instead this will additionally show an error message that provides more
+            // detailed information.
+            showLoggedErrorToUser(e, this.id, methodName)
```

Comment:
"to user" is never needed since that is implied by "show"
```suggestion
            showLoggedError(e, this.id, methodName)
```